src/modelbench/run.py

```diff
@@ -101,10 +103,13 @@ def benchmark(
     anonymize=None,
     parallel=False,
 ) -> None:
+    start_time = datetime.now(timezone.utc)
     suts = find_suts_for_sut_argument(sut)
-    benchmarks = [b() for b in BenchmarkDefinition.__subclasses__() if b.__name__ in benchmark]
-    benchmark_scores = score_benchmarks(benchmarks, suts, max_instances, debug, parallel)
+    benchmark = [b() for b in BenchmarkDefinition.__subclasses__() if b.__name__ == benchmark_name][0]
+    benchmark_scores = score_benchmarks([benchmark], suts, max_instances, debug, parallel)
     generate_content(benchmark_scores, output_dir, anonymize, view_embed, custom_branding)
+    json_path = output_dir / f"benchmark_record-{benchmark.uid}.json"
```

Comment:
Might be useful to provide an optional `--output-file` command option. In case a user wants to re-run a benchmark  without overwriting the last file.

Reply:
Interesting. I wanted it to be in a well-known location, so that other tools, like modellab, could pick it up. I was thinking that if they wanted to preserve an old run, they'd use the `--output-dir` argument to preserve the whole thing.

Reply:
Ah I forgot about `--output-dir`. Maybe that's good enough.